### PR TITLE
feat: on create alerts query range format is called to fill the variables

### PR DESCRIPTION
**Pull request by palashgdev**
Close https://github.com/SigNoz/engineering-pod/issues/1119

before: 

https://github.com/SigNoz/signoz/assets/88981777/74533ae1-7fe6-4847-b739-5965a9fa1182

after:

https://github.com/SigNoz/signoz/assets/88981777/a504c392-5a03-4373-873c-dc9e8ce56302

### CODE REVIEW

**Comment by srikanthccv** [comment]
@palashgdev https://www.loom.com/share/01277d5c3965484db1ddb7b777b7f454

**Review by srikanthccv — APPROVED** [review]
Tested

- Query builder in edit mode
- Query builder from dashboard panel expand options
- PromQL in edit mode
- PromQl dashboard panel expand options

File: frontend/src/hooks/queryBuilder/useCreateAlerts.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+				history.push(
+					`${ROUTES.ALERTS_NEW}?${QueryParams.compositeQuery}=${encodeURIComponent(
+						JSON.stringify(updatedQuery),
+					)}`,
+				);
+			},
+			onError: () => {
+				notifications.error({
+					message: SOMETHING_WENT_WRONG,
```
Comment (YounixM) [inline frontend/src/hooks/queryBuilder/useCreateAlerts.tsx:52]: Isn't the API responding with correct error message?

Comment (palashgdev) [reply frontend/src/hooks/queryBuilder/useCreateAlerts.tsx:52]: it is sending the error but I used the generic one for now... as the error format is not standardise yet...

Comment (palashgdev) [reply frontend/src/hooks/queryBuilder/useCreateAlerts.tsx:52]: can take up when it is standardised across frontend

**Comment by YounixM** [comment]
@palashgdev: Before and After recordings don't correspond to each other

**Comment by palashgdev** [comment]
> @palashgdev: Before and After recordings don't correspond to each other

@YounixM updated please check

**Review by Rajat-Dabade — APPROVED** [review]

**Comment by srikanthccv** [comment]
@YounixM please review.

File: frontend/src/hooks/queryBuilder/useCreateAlerts.tsx (hunk 1 above)
Comment (YounixM) [reply frontend/src/hooks/queryBuilder/useCreateAlerts.tsx:52]: If possible, let's use the error returned from API for now, we can take up standardising later as time permits.

**Review by YounixM — APPROVED** [review]

**Pull request merged**